dbms/src/Server/Server.cpp

```diff
@@ -1232,6 +1176,8 @@ int Server::main(const std::vector<std::string> & /*args*/)
     }
 
     if (global_context->getSharedContextDisagg()->isDisaggregatedStorageMode()
+        // disagg compute node without auto scaler will start proxy, which needs UniPS
+        || (global_context->getSharedContextDisagg()->isDisaggregatedComputeMode() && !global_context->getSharedContextDisagg()->use_autoscaler)
         || global_context->getSharedContextDisagg()->notDisaggregatedMode())
```

Comment:
How about simply remove this `if`?

Reply:
when it is deployed on tidb cloud with autoscaler, we don't need the `ps_wirte` instance :(

Reply:
simply by
https://github.com/pingcap/tiflash/blob/f1781b038fdc4e137160e7d1a9e2ccca97ce97d5/dbms/src/Server/Server.cpp#L1189-L1195